src/domain/errors.ts

```diff
@@ -11,13 +11,17 @@ export class UnknownRepositoryError extends RepositoryError {}
 export class PersistError extends RepositoryError {}
 export class DuplicateError extends RepositoryError {}
 
+export class UnknownPhoneProviderServiceError extends RepositoryError {}
```

Comment:
This should be in `src/domain/phone-provider/errors.ts` and extend `PhoneProviderServiceError`